macos.go

```diff
@@ -36,34 +36,36 @@ var (
 // createAccess creates a SecAccessRef as CFTypeRef.
 // The returned SecAccessRef, if non-nil, must be released via CFRelease.
 func createAccess(label string, trustedApplications []string) (C.CFTypeRef, error) {
-	if len(trustedApplications) == 0 {
```

Comment:
This will be true for either `nil` or `[]string{}`, won't this fail with either of those values and return a nil, which will mean "trust the calling application"?

Reply:
Ugh, ignore that, misread the diff.